proj/src/main/webapp/static/welcome.html

```diff
@@ -42,29 +42,29 @@ <h4>a spirit of friendly good-fellowship</h4>
             <div class="inblock-center-container">
               <h3 class="center-text"> meet friends <br> create communities</h3>
               <!-- hide button on desktop -->
-              <button type="button" name="button" class="button">begin</button>
+              <!-- <button type="button" name="button" class="button">begin</button> -->
```

Comment:
will this button come back when you start thinking about the mobile version or is it gone forever? if it's gone forever, I think you can remove it from the code.

Reply:
yup sorry i am doing a cleaning branch and removing excess code